pkg/uninstall/uninstall.go

```diff
@@ -80,6 +82,14 @@ func baseResources(ctx context.Context, c kclient.Client) (resources []kclient.O
 		}
 	}
 
+	controllerLease := &coordinationv1.Lease{}
+	if err := c.Get(ctx, kclient.ObjectKey{Namespace: "kube-system", Name: "acorn-controller"}, controllerLease); !apierror.IsNotFound(err) && err != nil {
+		return nil, err
+	}
+	if err == nil {
```

Comment:
the err here is not the err in the if statement above?

Reply:
You're right. Will fix.